api/tests/test_game_state_server.py

```diff
@@ -288,3 +289,27 @@ async def test_room_full(gss_with_client):
 
     with pytest.raises(InvalidConnectionException):
         gss_with_client.new_connection_request(TEST_CLIENT_ID, TEST_ROOM_ID)
+
+
+@pytest.mark.asyncio
+async def test_more_icons_than_colors(gss_with_client):
+    for i in range(len(colors) + 1):
```

Comment:
I think it would be valuable to have some kind of assertion here. Maybe just test that the last item gets placed w/out a color?